mmv1/products/compute/FutureReservation.yaml

```diff
@@ -545,3 +561,62 @@ properties:
         type: Integer
         description: |
           Span of time that's a fraction of a second at nanosecond resolution. Durations less than one second are represented with a 0 seconds field and a positive nanos field. Must be from 0 to 999,999,999 inclusive.
+  - name: 'enableEmergentMaintenance'
+    type: Boolean
+    description: |
+      Indicates if this group of VMs have emergent maintenance enabled.
+    immutable: true
+    ignore_read: true
```

Comment:
It doesn't look like this feature is fully rolled out yet, which is why `ignore_read` is necessary here - the attempt to set the field silently fails, so the field isn't returned in the response. This is also true for updates, but there the failure isn't silent, hence the need for `immutable`. Support for this field will need to be added in a future PR once the feature is complete (`aggregateReservation` is working just fine!)

Reply:
Sure, I removed `enableEmergentMaintenance` from the PR.